src/Raven.Client/Documents/Indexes/IndexProgress.cs

```diff
@@ -20,18 +20,17 @@ public class IndexProgress
 
         public class CollectionStats
         {
-            public long LastProcessedDocumentEtag { get; set; }
+            public long LastProcessedItemEtag { get; set; }
```

Comment:
This breaks backward compatibility. You can mark fields as obsolete, but the value should still be set in them.